### PR TITLE
v1.0-approval

### PR DESCRIPTION
**Pull request by igakigongo**

### CODE REVIEW

File: assets/css/992to1200/cover-articles.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
+.container {
+ display: grid;
+ grid-gap: 10px;
+ grid-template: repeat(2, 20em) / repeat(6, 1fr);
+ padding: 3px 5px;
+}
+
+.hero1 {
+ background-color: #000;
```
Comment (mcrd25) [inline assets/css/992to1200/cover-articles.css:9]: `background` would have been better and you could include lines 10, 11, 12 as well.

File: assets/css/992to1200/cover-articles.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+ grid-column: 1/5;
+ grid-row: 1/3;
+ opacity: 1;
+ position: relative;
+ transition: .4s ease;
+ will-change: opacity;
+}
+
+.hero2 {
+ background-image: url('../../images/hero-section/carrefour-blockchain-cryptocurrency-796x450.jpg');
```
Comment (mcrd25) [inline assets/css/992to1200/cover-articles.css:22]: same with line 9 comment

File: assets/css/992to1200/cover-articles.css [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+ background-position: 50%;
+ background-size: cover;
+ grid-column: 5/7;
+ grid-row: 1/2;
+}
+
+.hero3 {
+ background-color: gray;
```
Comment (mcrd25) [inline assets/css/992to1200/cover-articles.css:30]: Same with previous 2 hero classes and background

**Review by mcrd25 — COMMENTED** [review]
# WOW! ONE OF THE BEST OF THESE CLONES I'VE SEEN THUS FAR! 👍 💯 🥇. You may 🚀 

In general, your project looks great and It would not be right for me to request another review based on my review comments but I will, however, suggest a couple of things.
- Add a README (required). The README.md should include the name of the project,  your GitHub usernames, description to project and live version link to clone. You can get a markdown cheatsheet [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
- I noticed you handled media queries within HTML, which is fine, but probably best it is handled within a CSS file.
- You also have many CSS files, which you broke down I'm guessing for modularity which is awesome! I commend you for that. However, I suggest you should include them in a single file instead all in HTML file
- In general I realised you implemented hover effects for some of the links and not for tohers and some of the articles are clickable like original with cursor pointer effect and other are not. Maybe you can make it a more consistent clone by making all links look like links i.e see the pointer change when hover and add the hover effects to all elements of clone where applicable
- I noticed you did not do the coloured bullets, I am glad you  did not waste time doing it either, however, if you would like a tip on how to accomplish I'd be happy to help and guide

**Project really looks great. The only change you must implement is README.md. Once you add the README you may merge to master and 🚀. Project is soo well done 👍 💯!**

***If you have any questions, let me know here or on slack and I will be glad to clarify with you.
You are also free to disagree with me on points and create dialogue.***


**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/)

**Comment by igakigongo** [comment]
Hi, a little bit confused here with the suggestions regarding cover-articels.css, but we have also added the README.md file. Kindly look at [webpack-dev](https://github.com/igakigongo/The-Next-Web/tree/webpack-dev) branch as well, it's important with regards to the design decisions that we made.

**Pull request merged**